test/NuGet.Core.Tests/NuGet.Build.Tasks.Test/GetRestoreNuGetAuditSuppressionsTaskTests.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Build.Framework;
+using Xunit;
+
+namespace NuGet.Build.Tasks.Test
+{
+    public class GetRestoreNuGetAuditSuppressionsTaskTests
```

Comment:
I've added a duplicate test case in for now, since I haven't made any changes to the original task. If we decide to get rid of the task, or remove the duplication part of the task, I'll update/remove the tests accordingly.